test/gtest/uct/ib/test_dc.cc

```diff
@@ -561,6 +561,30 @@ UCS_TEST_P(test_dc_flow_control, fc_disabled_flush)
     test_flush_fc_disabled();
 }
 
+UCS_TEST_P(test_dc_flow_control, general_enabled_hw_dcs)
+{
+    modify_config("DC_MLX5_TX_POLICY", "hw_dcs");
```

Comment:
IMO better ass test variant to test_dc_flow_control to set HW_DCS policy
so that all tests in that suite will run with HW_DCS in addition to the default policy

Reply:
@yosefe  There's plenty of them. Are you sure we want to make `hw_dcs` a variant and parametrize everything with it? CI length..

Reply:
I think we need to have good coverage of hw_dcs, so yes
how much time does it add to the CI?

Reply:
9 tests * 4 policies = 36 added variations in total, increased from 33 to 69 tests, time diff is around 6 seconds

Reply:
extra 6s should be ok